schema/container.go

```diff
@@ -41,14 +45,16 @@ func (cm ContainerRuntimeManifest) MarshalJSON() ([]byte, error) {
 	return json.Marshal(containerRuntimeManifest(cm))
 }
 
+var cmKindError = types.InvalidACKindError(ContainerRuntimeManifestKind)
```

Comment:
I have inserted it as to DRY (to have the `"ContainerRuntimeManifest"` / `"ImageManifest"` strings defined in one place, and to enforce consistent error message by wrapping the error in `types.InvalidACKindError()`), and intent for the variable was to avoid instantiating same thing many times, using a private variable to avoid the accidental overwrite risk. If you find it overengineered and prefer calling `types.InvalidACKindError()` directly from the validation method, I'm fine with changing that (actually, Go compiler may be smart enough to not only inline the call, but realize it is actually a constant value, so there may be no real difference).